independent-publisher-connectors/Microsoft Graph Add Users/apiProperties.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{YOUR_CLIENT_ID}",
+          "scopes": [],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "{YOUR_REDIRECT_URL}",
+          "properties": {
+            "IsFirstParty": "True",

```

Comment:
May I know the reason for mentioning "True" for IsFirstParty?  First Party connectors are within Microsoft.